### DISCUSSION TITLE
Llama3.2-pth to bin

### DISCUSSION THREAD

**Discussion by zt672** [ggml-org/llama.cpp#9768] [Q&A] [answered]
I downloaded Llama3.2 from meta website and it is pth format. I notice that convert.py has deprecated and no pth_to_bin_convert.py now, so I wonder if anyone know how to convert now? Thank you so much!

**Comment by raulod** [comment, marked as answer]
you can use hf script `convert_llama_weights_to_hf.py` to convert from llama->hf->gguf but there are some issues when it comes to 3.2

https://github.com/huggingface/transformers/issues/33760
